app/views/admin/navigation_links/_form.html.erb

```diff
@@ -24,6 +24,20 @@
     <div data-svg-icon-upload-target="svgIconMessageValidate"></div>
   </div>
 </div>
+<div class="form-group">
+  <%= form.label :section %>
+  <div class="grid gap-4 grid-cols-2 l:grid-cols-3">
```

Comment:
accessibility (blocking) - to associate the radio buttons with the top-level "Section" label, we should group them in a [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) with the "Section" label being a `legend` element.

If we don't have these fieldset/legend elements, screen reader users won't know what the options "Default"/"Other" relate to. It's a bit of a special consideration for groups like radios, because even though the options have their own label, the whole group needs a label too 🙃